spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -134,7 +137,9 @@ private void registerSagaBeanDefinitions(Configurer configurer, BeanDefinitionRe
         for (String saga : sagas) {
             Saga sagaAnnotation = beanFactory.findAnnotationOnBean(saga, Saga.class);
             SagaConfiguration<?> sagaConfiguration = SagaConfiguration.subscribingSagaManager(beanFactory.getType(saga));
-            // TODO: Register resource injector here.
+
+            sagaConfiguration.configureResourceInjector(c -> beanFactory.getBean(ResourceInjector.class));
```

Comment:
The two lines were already replaced in the amended commits with this line that hopefully you didn't see, just before the loop:
```
configurer.configureResourceInjector(c -> getBean("resourceInjector", c));
```

I added a fixup commit (we'll rebase before merging), that takes even this one away, but as it is this requires the user to define the instance bean somewhere. I could not decide where to provide one by default.

Let me know, if there is anything else.